src/components/sbb-toggle-check/sbb-toggle-check.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { SbbToggleCheck } from './sbb-toggle-check';
+import { newSpecPage } from '@stencil/core/testing';
+
+describe('sbb-toggle-check', () => {
+  it('renders sbb-toggle-check', async () => {
+    const { root } = await newSpecPage({
+      components: [SbbToggleCheck],
+      html: '<sbb-toggle-check></sbb-toggle-check>',
+    });
+
+    expect(root).toEqualHtml(`
+        <sbb-toggle-check>
+          <mock:shadow-root>
+            <label class="toggle-check">
+              <input type="checkbox"/>
+              <span class="toggle-check__slider toggle-check__slider--after">
+                <span class="toggle-check__circle"></span>
+              </span>
+              <slot></slot>
+            </label>
+          </mock:shadow-root>
+        </sbb-toggle-check>
+      `);
+  });
+});
+
+describe('label position', () => {
```

Comment:
Move the describes into the `describe('sbb-toggle-check'` block, as that is the root block for all sbb-toggle-check tests